src/graphs/RenderGraph.jsx

```diff
@@ -0,0 +1,61 @@
+/** eslint verified */
+import React from 'react';
+import { ParentSize } from '@vx/responsive';
+import GraphLoader from './GraphLoader';
+
+/**
+ * Function to render a graph
+ *
+ * @param {any} data Graph data, it can be null (data hasn't loaded), false (data not available)
+ *  or an Object with the data.
+ * @param {string} labelX axis X label
+ * @param {string} labelY axis Y label
+ * @param {string} graph graph type
+ * @param {string} graphTitle graph title
+ * @param {array} colors colors to sort elements inside the graph
+ */
+const RenderGraph = (
+  data, labelX, labelY, graph, graphTitle, colors,
+  handlerInfoGraph, openInfoGraph, graphDescription, units,
+) => {
+  // While data is being retrieved from server
+  let errorMessage = null;
+  // (data === null) while waiting for API response
+  if (data === null) errorMessage = 'Cargando información...';
+  // (!data) if API doesn't respond
+  else if (!data) errorMessage = `Información${graphTitle ? ` de ${graphTitle}` : ''} no disponible`;
+  // (data.length <= 0) if API response in not object
```

Comment:
I think comments on lines 21, 23, 25 and 27 aren't necessary. But if you want to keep them, make them related wit the data itself, not the API, this component doesn't request an API, it receives data.

Reply:
Fixed on commit a590dc1